connectors/typescript/README.md

```diff
@@ -0,0 +1,22 @@
+# TypeScript Kalypso Connector
+
+This connector is based on NodeJS, TypeScript and Express. It is an example of a connector created based on the <a href="https://docs.kalypso.cloud/docs/partners/tutorial/" target="_blank">tutorial</a> in the docs.
+
+## File structure
+
+The connector is splitted in seperate folders:
+
+- `requests` contains the RequestHandlers for Actions and Triggers registrations
+- `store` contains the InMemory Subscription Store to save and remove registered subscriptions
+- `types` contains all required types
+
+## Running the connector
+
+To start the connector, run the following command:
```

Comment:
I guess the API Key for the connector must be provided first and added to an .env file or something like that

Reply:
I have added the API Key information